tests/test_tokenization_auto.py

```diff
@@ -310,6 +310,38 @@ def test_new_tokenizer_fast_registration(self):
             if CustomConfig in TOKENIZER_MAPPING._extra_content:
                 del TOKENIZER_MAPPING._extra_content[CustomConfig]
 
+    def test_from_pretrained_dynamic_tokenizer(self):
```

Comment:
Those are the new tests we can load a tokenzier with dynamic code, using the new and old "auto_map" markers.